src/Microsoft.AspNetCore.Http.Abstractions/Routing/IEndpointFeature.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Routing.Features
```

Comment:
Do we need this new namespace? This is the only thing in it.

Reply:
Other features are in a `.Features` namespace. On the other hand the current `IRoutingFeature` in `Microsoft.AspNetCore.Routing` isn't.

I don't have a strong opinion either way.

Reply:
@davidfowl Do you have a opinion here?

Reply:
I think it should go into the top level features namespace